src/screens/CreatePost/index.js

```diff
@@ -366,7 +366,6 @@ const CreatePost = () => {
       );
     }
   };
-
   const takePhoto = async () => {
     const {success, message} = await requestCameraPermission();
     if (success) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Extra indentation has been added here. This doesn't affect the functionality of the code but can impact readability. It's important to maintain consistent indentation throughout the codebase.
```diff
-    );
-  }
-};
-const takePhoto = async () => {
-  const {success, message} = await requestCameraPermission();
-  if (success) {
+    );
+    }
+  };
+  const takePhoto = async () => {
+    const {success, message} = await requestCameraPermission();
+    if (success) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->